translations/locale_es.ts

```diff
@@ -255,6 +255,16 @@ Subdirectories</source>
 Path</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <location filename="../batch_convert.qml" line="559"/>
+        <source>Filter files with</source>
+        <translation>Filter files with</translation>
```

Comment:
Than's not the spanish translation, better leave untranslated

Reply:
same for the other languages

Reply:
I don't get this one.

Reply:
Ok, don't worry, I'll fix it